src/Config/PluginConfig.php

```diff
@@ -0,0 +1,122 @@
+<?php
+namespace elanev\OpencastEvent\Config;
+
+use ActiveRecord;
+
+class PluginConfig extends ActiveRecord
```

Comment:
You don't know that, but the ActiveRecord will also be abolished in ILIAS at some point. Many developers have already started to switch their ActiveRecord implementations to the repository pattern. The main advantages are much better performance and you are no longer dependent on the AR service.

 this is also not something you have to do now, but I can show you the easiest way to replace this, the effort is usually not that great.